README.md

```diff
@@ -330,11 +330,25 @@ This creates a new metadata item of the specified type, name and value.
 
 This changes the value of an existing field, but will not change its type.
 
+You can't use `set()` to change core fields such as like `width` or
+`interpretation`.  Use `copy()` instead.`
```

Comment:
I think there's a stray backtick at EOL here

Reply:
Ups. Thanks for pointing it out.